<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectInputFromDataSet</class>
 <widget class="QWidget" name="SelectInputFromDataSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select input from dataset</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <item>
    <widget class="DataSetImageView" name="widgetDataSetImage" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSpinBox" name="spinBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSet">
       <property name="text">
        <string>Set image</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DataSetImageView</class>
   <extends>QWidget</extends>
   <header>DataSetImageView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
